apps/force/src/utils/index.ts

```diff
@@ -0,0 +1,230 @@
+import { gql, request } from "graphql-request";
+
+/**
+ * Map color values to one or more groups
+ *
+ * @param labels - An array containing one or more categorical values
+ * @param palette - An array containing a color scheme
+ *
+ * @returns Returns an object where the labels are mapped to colors
+ */
+
+export function createPalette(labels: Array, palette: Array): Object {
```

Comment:
Array of what type ? for example 'string []' or 'number []' or MyType[] ?

Reply:
I've removed the force dashboard from this PR as this should not be included in this PR.

In the latest commit, I fixed all typing issues.